### ISSUE TITLE
Issue warnings on TEST_MODE 

### ISSUE THREAD

**Issue by TamimiGitHub** [SolaceLabs/ep-asyncapi-importer-typescript#29]
## Overview

An example of warning would be if the content of an object changes in the spec file but the version already exists. When running this in RELEASE_MODE, object importing passes with a warning stating that the object version has been bumped based on the version bump strategy.

For example, after successful import of a spec, the importer outputs the following logs stating that there is a new version of the event and application. 

```
Import Summary for mode: release_mode

  Processed Application Domains: 1
    Created Application Domains:  0
  Processed Apis: 1
    Created Event Apis Versions:  1
    Created Application Verions:  1
  Processed Channels: 2
    Created Event Versions:       2
    Created Schema Versions:      1
    Created Enum Versions:        0

  Warnings: 2

[
  {
    "type": "VersionObjectWarning",
    "action": "CREATE_NEW_VERSION",
    "epObjectType": "eventApiVersion",
    "displayName": "ProcessPayment",
    "existingVersion": "0.0.2",
    "existingVersionState": "2",
    "targetVersion": "0.0.1",
    "targetVersionState": "2",
    "createdVersion": "0.0.3",
    "createdVersionState": "2",
    "runMode": "release",
    "applicationDomainName": "NYCTaxiTamimi",
    "timestamp": 1663778302350
  },
  {
    "type": "VersionObjectWarning",
    "action": "CREATE_NEW_VERSION",
    "epObjectType": "applicationVersion",
    "displayName": "ProcessPayment",
    "existingVersion": "0.0.2",
    "existingVersionState": "2",
    "targetVersion": "0.0.1",
    "targetVersionState": "2",
    "createdVersion": "0.0.3",
    "createdVersionState": "2",
    "runMode": "release",
    "applicationDomainName": "NYCTaxiTamimi",
    "timestamp": 1663778306598
  }
]
```

## Expected Behaviour

When running the importer in TEST_MODE, I would expect the importer to detect that there is a version mismatch and a new version will be created if the importer is run in RELEASE_MODE.

**Comment by jessemenning** [comment]
I would agree with this.  It would be very helpful, particularly with the GitHub action, where we want to prompt the committer to update versions.

**Comment by ricardojosegomezulmke** [comment]
implemented in release 0.9.0-alpha. compare release notes and documentation for exact behaviour. 
main difference: test pass will abort with error instead of a passing with a warning.
